plugin/evm/block_builder.go

```diff
@@ -169,11 +169,17 @@ func (b *blockBuilder) awaitSubmittedTxs() {
 					// Give time for this node to build a block before attempting to
 					// gossip
 					time.Sleep(waitBlockTime)
-					// [GossipTxs] will block unless [gossiper.txsToGossipChan] (an
-					// unbuffered channel) is listened on
-					if err := b.gossiper.GossipTxs(ethTxsEvent.Txs); err != nil {
+
+					// Retrieve nodes to gossip to
+					proposers, err := b.ctx.ProposerRetriever.GetCurrentProposers(context.TODO()) // UNSURE OF WHAT CONTEXT TO USE HERE
```

Comment:
`context.TODO()` is correct here since it indicates that we should in fact fill this in with a context later, but would need to refactor the code to make the correct context available to do so.

You can delete the comment for now and we have the `context.TODO()` to search for and replace with the correct context when we are ready to thread through all of the context's as needed.